### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by MarcGonzalezMoratona**
Fix Axolotl typos

Add new features to the readme description

### CODE REVIEW

**Review by gerard-queralt — APPROVED** [review]

**Review by DADSILVER — APPROVED** [review]

**Review by DADSILVER — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -93,6 +134,13 @@ You can drag and drop different models and edit it's meshes and textures (loadin
 **Editor Controller**
 - Play/Pause controller
 
+**Shortcuts**
```
Comment (MarcGonzalezMoratona) [inline README.md:137]: I know some of this shortcuts are not implemented yet, but I want to know which ones we can keep and which ones need to be added to a new draft PR @AuxPort66

File: README.md [hunk 2]
```diff
@@ -1,68 +1,109 @@
...
 - Link GameObjects to other GameObjects.
+- Reorder GameObjects in the scene hierarchy.
 
-**FILE SYSTEM**
+**File system**
 - The whole scene can be serialised and unserialized (any GameObject and its components can be serialised into binary Files).
-- Any scene can be save or load using an ImGui menu option.
-- The scene can be change into a JSON scene and loaded. 
-- Change an asset into the inspector the resource it's updated. 
-- Modify the save asset that's in a local folder is upgraded.
```
Comment (MarcGonzalezMoratona) [inline README.md:23]: I'm not sure if I understood properly this specific feature, can you check if it corresponds to this? @AuxPort66 

`- After modifying an asset from a local folder, the asset is updated automatically in the engine. `

File: README.md [hunk 3]
```diff
@@ -1,68 +1,109 @@
...
+- A scene can be transformed into a JSON file. 
+- Changing an asset in the inspector updated the corresponding resource.
+- After modifying an asset from a local folder, the asset is updated automatically in the engine. 
+- There's a GAME mode that loads all the files using a compressed zip of all the required assets and custom binary files. To activate this, just define a GAME predefined compilation variable in Globals.h.
 
 **Spatial Partitioning and Frustum Culling**
-- Build the Spatial Partitioning structure Based on QuadTree.
-- Have cull the objects/QuadTree that are inside the camera frustum.
+- Spatial Partitioning structure based on a QuadTree.
+- The engine culls the objects of the Quadtree that are outside the camera frustum.
```
Comment (MarcGonzalezMoratona) [inline README.md:32]: @Dedado3000 please review this change is correct

File: README.md [hunk 4]
```diff
@@ -1,68 +1,109 @@
...
 - Implementation of material maps (Diffuse, Specular and Shininess).
 - Interface features: Load and delete material maps, change diffuse and specular's color and shininess and normal's strength. 
 
 **Analytic lights and Normal map**
-- Different type of lights (Ambient, Directional, Point and Spot).
+- Different types of lights (Ambient, Directional, Point and Spot lights).
 - Lights can be added and removed and their parameters can be edited.
 - Application of the Normal Map and the possibility to edit it.
-- Application of the sky CubeMap. 
+- Implementation of the sky CubeMap.
```
Comment (MarcGonzalezMoratona) [inline README.md:47]: Here we could add something about HDR skybox. Feel free to add any suggestion @MacXxs

File: README.md [hunk 5]
```diff
@@ -1,68 +1,109 @@
...
 - Lights can be added and removed and their parameters can be edited.
 - Application of the Normal Map and the possibility to edit it.
-- Application of the sky CubeMap. 
+- Implementation of the sky CubeMap. 
+
+**Area lights**
+- Differents types of Area lights (Sphere, tube).
+
+**Breakable components**
+- Create a breakable component and edit its impulsion power.
```
Comment (MarcGonzalezMoratona) [inline README.md:53]: Feel free to add any suggestion you consider appropriate @AlexisDieu

File: README.md [hunk 6]
```diff
@@ -1,68 +1,109 @@
...
+
+**Physics**
```
Comment (MarcGonzalezMoratona) [inline README.md:55]: @Dedado3000 @OriolSors please check this set of features from physics and let me know if there's something you would change

File: README.md [hunk 7]
```diff
@@ -1,68 +1,109 @@
...
+- Ability to create rigidbodies and edit its properties.
+- Edit a Rigidbody collider and its properties (shape and transform).
+- Draw Debug options for Rigidbody colliders.
+- Add a proportional controller to a Rigidbody.
+
+**Particle systems**
+- Create a particle system and edit its properties (color, number of particles).
```
Comment (MarcGonzalezMoratona) [inline README.md:62]: @MacXxs I know the feature is in progress, but if you add here suggestions for the particle systems section I will bear them in mind for a draft PR that will be merged with your feature

File: README.md [hunk 8]
```diff
@@ -1,68 +1,109 @@
...
+
+**Audio**
+- Create and configure audio sources
```
Comment (MarcGonzalezMoratona) [inline README.md:65]: @UriMurcia add here any suggestion about audio implementation features that you consider relevant

File: README.md [hunk 9]
```diff
@@ -1,68 +1,109 @@
...
+
+**2D and 3D objects**
+- Create 2D canvas (Empty, Text, Image, Button).
+- Create 3D meshes (Cube, Sphere, Plane, Cylinder, Capsule, Character).
+
+**Animations and state machines**
```
Comment (MarcGonzalezMoratona) [inline README.md:71]: @AuxPort66 let me know if you want to add something else to the animation section

File: README.md [hunk 10]
```diff
@@ -1,68 +1,109 @@
...
+- Add an animation component to your meshes.
+- Ability to create, edit, rename and delete state machines.
+- Draw Debug options for sketelal animation bones.
+
+**Scripting**
```
Comment (MarcGonzalezMoratona) [inline README.md:76]: @gerard-queralt @markmeliaa @Piterga12 @DADSILVER  Add suggestions about features related to scripting that you consider relevant for the readme

File: README.md [hunk 11]
```diff
@@ -1,68 +1,110 @@
...
+**Physics**
+- Ability to create rigidbodies and edit its properties.
+- Edit a Rigidbody collider and its properties (shape and transform).
+- Draw Debug options for Rigidbody colliders.
+- Add a proportional controller to a Rigidbody.
+
+**Particle systems**
+- Create a particle system and edit its properties (color, color gradient, number of particles, duration, lifespan, size, rotation, gravity).
+- Add an emitter, select an emitter shape and customize properties (radius, angle).
+
```
Comment (MarcGonzalezMoratona) [inline README.md:64]:
```suggestion
- Add textures and spritesheet animations to a particle system.

```

File: README.md (hunk 4 above)
Comment (PaablooCH) [reply README.md:47]: hi, cubemap is not composed of 2 words it's just a single word

Comment (MarcGonzalezMoratona) [reply README.md:47]: thanks for noticing Pablo, I already solved that in another PR for the ImGui window too

File: README.md [hunk 12]
```diff
@@ -1,68 +1,111 @@
...
 - Implementation of material maps (Diffuse, Specular and Shininess).
 - Interface features: Load and delete material maps, change diffuse and specular's color and shininess and normal's strength. 
 
 **Analytic lights and Normal map**
-- Different type of lights (Ambient, Directional, Point and Spot).
+- Different types of lights (Ambient, Directional, Point and Spot lights).
 - Lights can be added and removed and their parameters can be edited.
 - Application of the Normal Map and the possibility to edit it.
-- Application of the sky CubeMap. 
+- Implementation of the sky CubeMap.
```
Comment (MarcGonzalezMoratona) [inline README.md:47]:
```suggestion
- Implementation of the sky cubemap. 
```

File: README.md (hunk 2 above)
Comment (AuxPort66) [reply README.md:23]: It's correct. 
You can also add that we have a resource inspector when you can change Import and load options of the resource.
And on the GAME mode section, now we have a build button in the engine where you can create the build and the zip

File: README.md (hunk 9 above)
Comment (AuxPort66) [reply README.md:73]: Instead of "Ability to create, edit, rename and delete state machines"
- Full state machine visual editor where you can create transitions and states with animations and control them on run time with scripts. Accessible through the animation component

File: README.md (hunk 1 above)
Comment (AuxPort66) [reply README.md:139]: Q/W/E on scene window to change operation type of the gyzmos

File: README.md [hunk 13]
```diff
@@ -1,68 +1,109 @@
...
+
+**Animations and state machines**
+- Add an animation component to your meshes.
+- Ability to create, edit, rename and delete state machines.
+- Draw Debug options for sketelal animation bones.
+
+**Scripting**
+- Ability to create and link scripts to GameObjects.
+
+**Build**
```
[the remark this answers is not in the document]
Comment (gerard-queralt) [reply README.md:81]: I'd maybe reword it, if you don't mind my suggesting.
- Specify that you are _choosing_ a starting scene, in my opinion "configure" is too broad of a term.
- This is subjective, but to me "environment" sounds server-related, I think "configuration" is a better word.
- A "zip" is a type of file, not a folder

This is my suggestion, but I trust on your verbose over mine :P
```Markdown
Use the `Build` button in the graphical interface to select the starting scene of your game, choose the build configuration and create a `.zip` file with all the assets and resources necessary to run the resulting executable file.
```

File: README.md (hunk 10 above)
Comment (markmeliaa) [reply README.md:79]: - Create your own scripts from the Inspector.
- Support a great amount of primitives in the script, and make them visible or not in the Inspector.

(And things like hot-reloading or other stuff should be added, but idk if they are working properly yet, @gerard-queralt will know).

File: README.md (hunk 7 above)
Comment (MacXxs) [reply README.md]: Some of the editable parameters are: velocity, gravity, size, rotation, emitter shape, spawnrate, texture (with tiling, blending mode, and blending speed), color (and alpha value) over time, rotation over time, size over time, alignment, duration of the particle simulation, looping. Great part of them also can be initialized with a random value between a range set by the user

**Review by AuxPort66 — APPROVED** [review]

**Review by Piterga12 — APPROVED** [review]

**Review by markmeliaa — APPROVED** [review]
